vortexasdk/endpoints/cargo_movements.py

```diff
@@ -40,6 +40,7 @@ def search(
         filter_storage_locations: Union[ID, List[ID]] = None,
         filter_ship_to_ship_locations: Union[ID, List[ID]] = None,
         filter_waypoints: Union[ID, List[ID]] = None,
+        exclude: Dict[ID, Union[ID, List[ID]]] = None,
```

Comment:
Actually that's the most important change. Exclusion filter must be a `dict` where keys are the parameters we want to apply the exlusions (e.g. `filter_products`, `filter_origins` etc.) and values are the corresponding IDs. I attach an image to visualize a relevant POST request in the UI.

![exclude_query_POST](https://user-images.githubusercontent.com/31421156/80430610-fe635600-88e6-11ea-85db-29e750592fce.PNG)